var/spack/repos/builtin/packages/py-platformdirs/package.py

```diff
@@ -13,14 +13,17 @@ class PyPlatformdirs(PythonPackage):
     homepage = "https://github.com/platformdirs/platformdirs"
     pypi = "platformdirs/platformdirs-2.4.0.tar.gz"
 
+    version("2.5.2", sha256="58c8abb07dcb441e6ee4b11d8df0ac856038f944ab98b7be6b27b2a3c7feef19")
     version("2.4.1", sha256="440633ddfebcc36264232365d7840a970e75e1018d15b4327d11f91909045fda")
     version("2.4.0", sha256="367a5e80b3d04d2428ffa76d33f124cf11e8fff2acdaa9b43d545f5c7d661ef2")
     version("2.3.0", sha256="15b056538719b1c94bdaccb29e5f81879c7f7f0f4a153f46086d155dffcd4f0f")
 
     depends_on("python@3.7:", when="@2.4.1:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"))
-    depends_on("py-setuptools@44:", type="build")
-    depends_on("py-setuptools-scm@5:+toml", type="build")
+    depends_on("py-setuptools@44:", when="@:2.5", type="build")
+    depends_on("py-setuptools-scm@5:+toml", when="@:2.5", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@44:", when="@:2.5.1", type="build")
    depends_on("py-setuptools-scm@5:+toml", when="@:2.5.1", type="build")
```
`@:2.5` includes 2.5.2